### ISSUE TITLE
Not working in iOS and Firefox

### ISSUE THREAD

**Issue by johandrott** [MartijnW49/ngx-image-cropper#12]
Hi

I can't get the cropper to work with Firefox and iOS-devices, all the resize dots are in the same position and it's not possible to move them. See screenshot below

![skarmavbild 2018-01-22 kl 14 56 43](https://user-images.githubusercontent.com/6651090/35224633-f120580e-ff85-11e7-84b6-e481299305e7.png)

I have configured the cropper like so:   
<image-cropper
        [imageChangedEvent]="imageChangedEvent"
        [maintainAspectRatio]="true"
        [aspectRatio]="16 / 9"
        [resizeToWidth]="1200"
        format="jpeg"
        (imageCropped)="imageCropped($event)"
    ></image-cropper>

**Comment by johandrott** [comment]
I found that this is a timing issue. The image is not loaded when calculating the cropper position. I solved this by adding a load event to the img tag and added a function that resets the cropper position after the image is loaded. This also removes the need for the timeout function used to reset the cropper.
```

 <img
        [src]="imgDataUrl"
        [style.visibility]="imageVisible ? 'visible' : 'hidden'"
        (load)="imageDataUrlLoaded()"
        class="source-image"
    />
```

```
 public imageDataUrlLoaded(): void {
        this.resetCropperPosition();
        this.imageLoaded.emit();
    }
```

**Comment by MartijnW49** [comment]
Hi Johandrott, thanks for the fix! I added it in a new release: 0.1.17
